src/main/java/org/fungover/haze/Main.java

```diff
@@ -74,14 +74,25 @@ private static void printThreadDebug() {
     }
 
     public static String executeCommand(HazeDatabase hazeDatabase, List<String> inputList) {
+        if (inputList == null || inputList.isEmpty() || inputList.get(0) == null || inputList.get(0).isEmpty()) {
```

Comment:
```suggestion
        if (inputList.isEmpty() || inputList.get(0).isEmpty()) {
```

Reply:
If i change the code, I won't be able to get all the test through (green).

Reply:
Yes you are :stuck_out_tongue_winking_eye:

Reply:
Did you thinking like this:
```java
       try {
            commandEnum = Command.valueOf(command);

            return switch (commandEnum) {
                case PING -> hazeDatabase.ping(inputList);
                case SETNX -> hazeDatabase.setNX(inputList);
                case SAVE -> SaveFile.writeOnFile(hazeDatabase.copy());
                case SET -> null;
                case GET -> null;
                case DEL -> hazeDatabase.delete(inputList.subList(1, inputList.size()));

                case EXISTS -> null;
            };
        } catch (IllegalArgumentException ex) {
            return "-ERR unknown command\r\n";
        }
```

Reply:
> Yes you are stuck_out_tongue_winking_eye

My bad